src/generators/csharp/renderers/ClassRenderer.ts

```diff
@@ -100,14 +100,22 @@ export const CSHARP_DEFAULT_CLASS_PRESET: CsharpClassPreset<CSharpOptions> = {
     return renderer.defaultSelf();
   },
   async property({ renderer, property, options }) {
+
+    let nullablePropertyEnding = '';
+    if (options?.handleNullable && !property.required) {
+      nullablePropertyEnding = '= null!';
```

Comment:
I would change this to
```suggestion
      nullablePropertyEnding = ' = null!';
```

Reply:
Fixed in latest commit